src/views/Login.vue

```diff
@@ -228,71 +236,4 @@
   })
   export default class Login extends Vue {}
 </script>
-<style scoped lang="scss">
```

Comment:
why everything was removed ? If it was to re-use the style of `Login` component in `ResetPassword`, well please change the `id` and use class `selector` first and don't copy/move everything.